src/scripts/validate-schema-updates.sh

```diff
@@ -0,0 +1,18 @@
+differences=$(git diff --name-only src/schemas/)
+
+if [ "$differences" != "" ]
+then
+        tput setaf 1;
+        echo "The following JSON Schemas have been updated:"
+        tput setaf 3;
+        echo "    ${differences}"
+        tput setaf 1;
+        echo "Please review and commit the changes."
+        tput sgr0;
+        exit 1
+else
```

Comment:
I wonder if it would be preferable to not generate any output if there are no changes, just to reduce noise. What do you think?

Reply:
In the same way we see green text in the terminal saying our tests have passed, I thought we'd like to see the same type of output for a task like this one. But I don't have a preference either way. Seeing the green text over time might help us remember to run the generate task before we even commit the work.